src/utils/addAtom.js

```diff
@@ -0,0 +1,25 @@
+import * as THREE from "three";
+
+/**
+ * addAtom
+ *
+ * @type {Function} Adds new atom to the scene.
+ *
+ * @param {Object} scene
+ * @param {Array} atomCoor
+ * @param {Number} atomRadius
+ * @param {Array} atomColor
+ *
+ */
+
+const addAtom = (scene, atomCoor, atomRadius, atomColor) => {
+  const atomGeo = new THREE.SphereGeometry(atomRadius, 8, 8);
+  const rgbColor = new THREE.Color();
+  rgbColor.setRGB(atomColor[0] / 255, atomColor[1] / 255, atomColor[2] / 255);
```

Comment:
I think the radius and color should be customizable in the future. The user should be able to overwrite the default values for each atom.